### PR TITLE
Fixed JS asset hash is not in sync with PHP one.

### PR DESCRIPTION
**Pull request by fabianmarz**
### Description
- This PR fixes the JS asset hash to be in sync with the one generated from the PHP function
- While the JS one was using `?_=<md5hash>` the PHP one was using `?v=<filemtime>`
- This requires https://github.com/netzstrategen/twig-asset/pull/2 to work

### CODE REVIEW

File: tasks/styles.js [hunk 1]
```diff
@@ -55,6 +57,9 @@ module.exports = (gulp, $, pkg) => {
       .pipe($.if(options.production, $.rename({ suffix: '.min' })))
       .pipe($.if(options.concat, $.concat(pkg.title.toLowerCase().replace(/[^a-z]/g,'') + '.css')))
       .pipe($.cssUrlCustomHash({
+        customHash: (fileName, hash, filePath) => {
+          return path.basename($.twigAsset.asset(filePath));
+        },
```
Comment (sun) [inline tasks/styles.js:62]: This code doesn't change the parameter name from `_` to `v` yet?  Is that the remaining todo?

Comment (fabianmarz) [reply tasks/styles.js:62]: 🙈 seems I missed it… thanks

File: package.json [hunk 2]
```diff
@@ -12,13 +12,14 @@
...
     "@netzstrategen/fractal-twig": "^2.2.4",
+    "@netzstrategen/twig-asset": "^1.0.0",
     "del": "^3.0.0",
     "gulp": "^4.0.0",
     "gulp-autoprefixer": "^4.0.0",
     "gulp-babel": "^8.0.0",
     "gulp-clean-css": "^3.7.0",
     "gulp-concat": "^2.6.1",
-    "gulp-css-url-custom-hash": "^0.0.5",
+    "gulp-css-url-custom-hash": "netzstrategen/gulp-css-url-custom-hash",
```
[the remark this answers is not in the document]
Comment (fabianmarz) [reply package.json:22]: Yes, as the fork of this package supports getting file information (file mtime) via the filepath.

File: tasks/styles.js (hunk 1 above)
Comment (fabianmarz) [reply tasks/styles.js:62]: Ah, now I see. The query string is returned by the `$.twigAsset.asset()` function

**Comment by fabianmarz** [comment]
Ok thanks. Going to have a look into it again.

**Review by wrux — APPROVED** [review]

**Pull request merged**